release-notes/5.0/README.md

```diff
@@ -2,5 +2,6 @@ The following Microsoft.Data.SqlClient 5.0 preview releases have been shipped:
 
 | Release Date | Version | Notes |
 | :-- | :-- | :--: |
+| 2022/06/15 | 5.0.0-preview3 | [release notes](5.0.0-preview3.md) |
```

Comment:
this section needs to be updated after the release when we have the exact date and  number of build available.